README.md

```diff
@@ -1 +1,44 @@
-# MultiDID
+# multidid
+
+> A very compact representation of DIDs and DID URLs
+
+Multidid is a representation strategy for DIDs and DID URLs that is very compact and extensible. It allows any DID method to be represented as a string of bytes.
+
+## Format
+
+```
+<multidid-code><method-code><method-bytes><url-length><url-bytes>
+```
+
+Where:
+
+* `multidid-code` - the value `0x0d1d` encoded as a varint
```

Comment:
which kind of varint is this? LEB128 or multiformats unsigned-varint?

Reply:
Multiformats